node/src/main/scala/org/bitcoins/node/networking/peer/DataMessageHandler.scala

```diff
@@ -209,7 +209,9 @@ case class DataMessageHandler(
                        "which is less than max. This means we are synced,",
                        "not requesting more.")
                     .mkString(" "))
-                if (appConfig.isNeutrinoEnabled && !syncing)
+                if (
```

Comment:
since this is kind of a nasty expression, can you just put a comment above the `if` saying what this checking in plain english?

Reply:
maybe link to the issue in source code too?